tests/linearGaussian_snpe_test.py

```diff
@@ -383,7 +379,7 @@ def simulator(theta):
 
 
 # Testing rejection and mcmc sampling methods.
-@pytest.mark.slow
+# @pytest.mark.slow
```

Comment:
great, and the tests pass! Please re-add the comment for these tests and we can merge!